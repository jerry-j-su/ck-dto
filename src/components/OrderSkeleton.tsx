/**
 * If order list is held rendering due to performance, Skeletons will be rendered until further content nodes are added
 */
export default function OrderSkeleton() {
    return (
        <>
            <div className="skeleton-title" />
            <div className="skeleton-line" />
            <div className="skeleton-line" />
            <div className="skeleton-line" />
        </>
    )
}
