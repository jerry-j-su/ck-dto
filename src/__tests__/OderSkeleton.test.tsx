import OrderSkeleton from "../components/OrderSkeleton";

test('<OrderSkeleton Basic Rendering', () => {
    expect(OrderSkeleton()).toMatchSnapshot()
})
