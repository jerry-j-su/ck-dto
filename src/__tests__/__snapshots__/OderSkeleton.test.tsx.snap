// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrderSkeleton Basic Rendering 1`] = `
<React.Fragment>
  <div
    className="skeleton-title"
  />
  <div
    className="skeleton-line"
  />
  <div
    className="skeleton-line"
  />
  <div
    className="skeleton-line"
  />
</React.Fragment>
`;
